troubleshooting_central/backing-up-central-database-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="backing-up-central-database-by-using-the-roxctl-cli"]
+= Backing up Central database by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: backing-up-central-database-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+Central stores information about:
```

Comment:
Is it common in this doc set to just refer to the db by name only? At least in a first instance per assembly (or maybe per module), I might expect something like:
```suggestion
The Central database stores information about the following:
```
Also IBM SG says avoid having a lead-in sentence continue into a list.